Sources/SwiftDocC/Semantics/Metadata/DocumentationExtension.swift

```diff
@@ -47,6 +49,25 @@ public final class DocumentationExtension: Semantic, AutomaticDirectiveConvertib
         case override
     }
     
+    func validate(source: URL?, for bundle: DocumentationBundle, in context: DocumentationContext, problems: inout [Problem]) -> Bool {
+        if behavior == .append {
+            let diagnostic = Diagnostic(
+                source: source,
+                severity: .information,
```

Comment:
Should this be `.warning` instead? In the PR description you mentioned that DocC would emit a warning. However, it doesn't. Since by default DocC does not emit information diagnostics, DocC users will never see this message.

Related:
- How does a cmd line user specify the diagnostic level (i.e. that they want to see information or debug messages)?
- What diagnostic level does Xcode request when it launches DocC?

Reply:
Good catch. I intended for this to be a warning.